ipa.if

```diff
@@ -308,3 +308,62 @@ interface(`ipa_read_tmp',`
 
     read_files_pattern($1, ipa_tmp_t, ipa_tmp_t)
 ')
+
+## <summary>policy for ipa_custodia</summary>
```

Comment:
I suppose the original header can be omitted here.

Reply:
Sure, I thought it would denote "ipa_custodia" section, but apparently that is only done in ".te" files. I'll remove it.